apps/vaporgui/CMakeLists.txt

```diff
@@ -106,6 +105,42 @@ set (SRCS
 	VSection.h
 	VLineItem.cpp
 	VLineItem.h
+    VContainer.cpp
+    VContainer.h
+    VComboBox.cpp
+    VComboBox.h
```

Comment:
There's an annoying behavior of QComboBox, which responds to the mouse wheel. Could VComboBox disable that? 
![Dec-12-2019 16-46-59](https://user-images.githubusercontent.com/814885/70758182-55350e00-1cff-11ea-8639-0eaf562c77c6.gif)

Reply:
Fixed